libbeat/logp/log.go

```diff
@@ -136,7 +136,16 @@ func Critical(format string, v ...interface{}) {
 // The version string contains the version when the future willb e removed
 func Deprecate(version string, format string, v ...interface{}) {
 	postfix := fmt.Sprintf(" Will be removed in version: %s", version)
-	msg(LOG_WARNING, "WARN DEPRECATED: ", format+postfix, v...)
+	Warn("DEPRECATED: "+format+postfix, v...)
+}
+
+// Experimental is used to log experimetan
```

Comment:
Seems like this comment is incomplete. And it has a type. Suggest - "Experimental logs the usage of an experimental feature."